### ISSUE TITLE
Incorrect test in Psychrometic function: GetTDewPointFromVapPres().

### ISSUE THREAD

**Issue by drajmarsh** [psychrometrics/psychrolib#6]
The following range check at line 285 in the file 'psychrolib/src/js/psychrolib.js' appears to incorrectly use `&&` rather than `||`, or should be surrounded by a `!()`.

EXISTING CODE:
```
  // Bounds outside which a solution cannot be found
  if (VapPres <= this.GetSatVapPres(_BOUNDS[0]) && VapPres >= this.GetSatVapPres(_BOUNDS[1]))
    throw new Error("Partial pressure of water vapor is outside range of validity of equations");
```

PROBABLE INTENDED CODE:
```
  // Bounds outside which a solution cannot be found
  if (!(VapPres <= this.GetSatVapPres(_BOUNDS[0]) && VapPres >= this.GetSatVapPres(_BOUNDS[1])))
    throw new Error("Partial pressure of water vapor is outside range of validity of equations");
```
As _BOUNDS[0] is always less than _BOUNDS[1], the existing check will not work.

---------

Full path to the file being referenced in this bug report:
https://github.com/psychrometrics/psychrolib/blob/59d085b2df276578b4ead6b1fd5c8d6ce0e91520/src/js/psychrolib.js

**Comment by dmey** [comment]
@drajmarsh many thanks for reporting this to us! This was indeed a mistake and have been fixed in #7 -- I have opted to use the logical OR instead as done in the other implementations. 
Once #7 is merged into master, it will be a few weeks before we will make the official release therefore please use the one in master until then. I will close this issue once #7 has been merged into master.
